src/tools/perf/perftest.c

```diff
@@ -42,13 +42,13 @@ test_type_t tests[] = {
      "get latency / bandwidth / message rate", "latency", 1},
 
     {"fadd", UCX_PERF_API_UCT, UCX_PERF_CMD_FADD, UCX_PERF_TEST_TYPE_STREAM_UNI,
-     "atomic fetch-and-add latency / rate", "latency", 1},
+     "atomic fetch-and-add latency / rate", "latency", 32},
 
     {"swap", UCX_PERF_API_UCT, UCX_PERF_CMD_SWAP, UCX_PERF_TEST_TYPE_STREAM_UNI,
-     "atomic swap latency / rate", "latency", 1},
+     "atomic swap latency / rate", "latency", 32},
 
     {"cswap", UCX_PERF_API_UCT, UCX_PERF_CMD_CSWAP, UCX_PERF_TEST_TYPE_STREAM_UNI,
-     "atomic compare-and-swap latency / rate", "latency", 1},
+     "atomic compare-and-swap latency / rate", "latency", 32},
```

Comment:
There are also other atomic operations, not affected by this PR. Is that intentional? Yes / No / explain...

Reply:
Intentional, those are the only atomic operations that are waiting for responses:
![image](https://user-images.githubusercontent.com/19361875/235429266-8452699b-46c2-47b3-92a4-04dbc1e78bcb.png)
![image](https://user-images.githubusercontent.com/19361875/235429297-ca446428-8fa5-4b62-8f51-bfce83f42275.png)